documentation/modules/non-admin-permissions-for-ui.adoc

```diff
@@ -0,0 +1,72 @@
+:_content-type: CONCEPT
+
+[id="non-admin-permissions_{context}"]
+= Permissions needed by non-administrators to work with migration plan components
+
+If you are an administrator, you can work with all components of migration plans (for example, providers, network mappings, and migration plans).
+
+By default, non-administrators have limited ability to work with migration plans and their components. As an administrator, you can modify their roles to allow them full access to all components for the namespaces they have access to, or you can give them limited permissions.
```

Comment:
```suggestion
By default, non-administrators have limited ability to work with migration plans and their components. As an administrator, you can modify their roles to allow them full access to all components, or you can give them limited permissions.
```
(the example that follows includes cluster-roles, so let's skip the per-namespace permissions here)